### PR TITLE
Move svn state tests to pytest

### PR DESCRIPTION
**Pull request by krionbsd**
### What does this PR do?
Move svn state tests to pytest

### CODE REVIEW

File: tests/pytests/unit/states/test_svn.py [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+                        "result": True,
+                    }
+
+
+def test_latest_trust_failures():
+    """
+        Test that checks that the trust_failures option is handled
+        correctly when running svn.latest in test mode. This tests for the
+        bug reported as #59069.
+    """
```
Comment (s0undt3ch) [inline tests/pytests/unit/states/test_svn.py:67]:
```suggestion
    """
    Test that checks that the trust_failures option is handled
    correctly when running svn.latest in test mode. This tests for the
    bug reported as #59069.
    """
```

File: tests/pytests/unit/states/test_svn.py [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+            "42:HEAD",
+            "--trust-server-cert-failures",
+            "unknown-ca",
+        )
+
+
+def test_export():
+    """
+        Test to export a file or directory from an SVN repository
+    """
```
Comment (s0undt3ch) [inline tests/pytests/unit/states/test_svn.py:96]:
```suggestion
    """
    Test to export a file or directory from an SVN repository
    """
```

File: tests/pytests/unit/states/test_svn.py [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+                    "comment": True,
+                    "name": "salt",
+                    "result": True,
+                }
+
+
+def test_dirty():
+    """
+        Test to determine if the working directory has been changed.
+    """
```
Comment (s0undt3ch) [inline tests/pytests/unit/states/test_svn.py:138]:
```suggestion
    """
    Test to determine if the working directory has been changed.
    """
```

**Review by s0undt3ch — CHANGES_REQUESTED** [review]

File: tests/pytests/unit/states/test_svn.py (hunk 3 above)
Comment (krionbsd) [reply tests/pytests/unit/states/test_svn.py]: Fixed

**Review by s0undt3ch — APPROVED** [review]

**Pull request merged**